en/app.ftl

```diff
@@ -394,6 +398,22 @@ profile-filter-category-option-domain-based-aliases-v2 = Custom aliases
 # Filter on Relay aliases that only forward critical emails.
 profile-filter-category-option-critical-only-aliases = Critical-only aliases
```

Comment:
@maxxcrawford Hmm, looks like this string was already there, but it's the only place we refer to "critical" emails - maybe check with Chris?